src/js/containers/App/App.js

```diff
@@ -26,34 +26,45 @@ class AppContainer extends Component {
-                        <LayoutColumn width={1/3}>&nbsp;</LayoutColumn>
-                        <LayoutColumn width={1/3}>
-                            <img src={ getAbsoluteUrl(config, 'assets/logo.svg') } />
-                        </LayoutColumn>
-                        <LayoutColumn width={1/3}>&nbsp;</LayoutColumn>
-                    </LayoutRow>
-                    <LayoutRow>
-                        <LayoutColumn width={1/4}>
-                            { isLoggedIn() ? <ActiveZoneSelector/> : <noscript/> }
-                        </LayoutColumn>
-                        <LayoutColumn width={2/4}>&nbsp;</LayoutColumn>
-                        <LayoutColumn width={1/4}>
-                            { (isLoggedIn() && this.props.state.zoneSettings.entities[this.props.state.activeZone.id]) ? <UnderAttackButton/> : <noscript/> }
-                        </LayoutColumn>
-                    </LayoutRow>
-                    <LayoutRow>
-                        <LayoutColumn width={1/3}>&nbsp;</LayoutColumn>
-                        <LayoutColumn width={1/3}>
-                            <AppNavigation />
-                        </LayoutColumn>
-                        <LayoutColumn width={1/3}>&nbsp;</LayoutColumn>
+                        <header id="header" className="header app-header">
+                            <div className="gradient-bar-header">
+                                <div className="header-main" style={{ borderBottom: "1px solid #ddd", backgroundColor: "#f9f9f9" }}>
+                                    <LayoutColumn width={3/8}>
+                                        <LayoutColumn width={3/8}>
+                                            <img style={{ margin: "4.95px auto 0", width: "170px", height: "30px" }} src={ getAbsoluteUrl(config, 'assets/logo.svg') } />
+                                        </LayoutColumn>
+                                        <LayoutColumn width={5/8}>
+                                            { isLoggedIn() ? <ActiveZoneSelector/> : <noscript/> }                                            
+                                        </LayoutColumn>
+                                    </LayoutColumn>     
+                                    <LayoutColumn width={5/8}>
+                                        <div style={{ float: "right" }}>
+                                            { (isLoggedIn() && this.props.state.zoneSettings.entities[this.props.state.activeZone.id]) ? <UnderAttackButton/> : <noscript/> }
+                                        </div>
+                                    </LayoutColumn>
+                                </div>
+                            </div>
+                        </header>
                     </LayoutRow>
+
+                    <div style={{ marginTop: "-8px" }}>
+                        <LayoutRow>
+                            <div style={{ display: "flex", padding: "1rem", backgroundColor: "#FFF" }}>
```

Comment:
We have [cf-component-flex](https://cloudflare.github.io/cf-ui/#cf-component-flex) for this, we should never be writing `display: "flex";`.  Why doesn't `<LayoutRow>` by itself work here?  Our inline styles should be page / component specific and not structure related.